stardog/values.yaml

```diff
@@ -23,6 +23,20 @@ stardog:
   users:
   # - name: nameOfTheUser
   #   password: passwordForTheUser
+  disabledUsers:
+  - anonymous
+  databases:
+  # - name: stardog
+  #   options:
+  #   - name: option
+  #     value: value
+  roles:
```

Comment:
We could create a default `superuser` role to be used: https://www.stardog.com/docs/#_superusers

Reply:
Hm, a superuser role is already present internally. This can be achieved easily with the superuser flag at user creation, e.g. `bash ${script} -v --ensure-user {{ .name }} --user-pass-from "/run/secrets/stardog/{{ .name }}" -- --superuser` would result in `${admin_bin} user add --new-password $(cat "${user_pass_from}") --passwd "${admin_pw}" "${user_name}" --superuser`. Should we can add the flag in the `.users` dict or create/provide a new role out of it?

Reply:
Let's try it with the role, this way the superuser is just a regular role (and could be changed after user creation)